src/base/static/components/input-form/input-form-category-button.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import cx from "bem-classnames";
+
+import { LabelWithInlineImage } from "../ui-elements/label-with-inline-image";
+
+const baseClass = "input-form-category-button";
+
+class InputFormCategoryButton extends Component {
+
+  constructor() {
+    super(...arguments);
```

Comment:
In general, I think it's best practice to avoid having to access the `arguments` global directly. In this case, we can use React props like so:

```
constructor(props) {
   super(props)
```

See here: https://reactjs.org/docs/react-component.html#constructor

Reply:
Ah, good point.